package.json

```diff
@@ -14,6 +14,7 @@
     "@babel/preset-typescript": "^7.18.6",
     "@babel/runtime": "^7.20.1",
     "@trussworks/react-uswds": "^3.2.0",
+    "@types/jest": "^29.5.1",
```

Comment:
This package was added to fix some warnings from TypeScript and is unrelated to the pagination work.